### DISCUSSION TITLE
waybar font

### DISCUSSION THREAD

**Discussion by tsuki-xi77** [prasanthrangan/hyprdots#310] [Q&A]
i'm really struggling to set a different font in waybar and hoping someone can tell me what i'm missing. i successfully changed gtk, qt, kitty fonts... but only reference i've found to waybar's font is in .config/waybar/style.css, it always reverts back to JetBrains when i reload waybar to see the change. Is this font being set somewhere else I haven't found? my eyes are old on a hidpi screen and want to use a wider more legible font. Sorry if this is an obvious answer- i'm really new to window managers. Also many thanks for this amazing repo, you are appreciated!

**Comment by tsuki-xi77** [comment]
finally found it. i'll leave this here in case someone else gets stuck. 

.config/waybar/modules/style.css

**Reply by Muhammad-Abubakr** [reply]
Thanks, Man. I owe you one.
